docs/DeveloperGuide.md

```diff
@@ -879,108 +896,262 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
```

Comment:
Actually it works to put everything as "1", it will automatic render the ascending number

Reply:
Nvm i change already